src/ET_Client.php

```diff
@@ -657,10 +657,12 @@ function getAuthTokenExpiration($tenantKey)
 	*/
 	function getInternalAuthToken($tenantKey) 
 	{
-		$tenantKey = $tenantKey == null ? $this->tenantKey : $tenantKey;	
-		if ($this->tenantTokens[$tenantKey] == null) {
-			$this->tenantTokens[$tenantKey] = array();
-		}
+    $tenantKey = $tenantKey == null ? $this->tenantKey : $tenantKey;
+    
+    if (!isset($this->tenantTokens) || !isset($this->tenantTokens[$tenantKey])) {
+      return null;
+    }
+
```

Comment:
Argh... tabs vs. spaces. Also, this whole method can be reduced to 2 lines now :wink:
```suggestion
		$tenantKey ??= $this->tenantKey;

		return $this->tenantTokens[$tenantKey]['internalAuthToken'] ?? null;
```
Beware applying this recommendation will leave the original `return` statement below; if applied, the statement below should be removed as it will no longer be reached.